blacksauce/src/_includes/_header.html

```diff
@@ -0,0 +1,23 @@
+<div class="grid--nav u-black u-marginNone">
+<img class="logo_nav" src="/images/logo.png" alt="blacksauce logo">
+
+<nav class="wrapper site-nav">
+ {# @feedback there is a red = sign but I can't figure out why #}
+		<button id="js-nav-trigger" aria-controls="js-site-nav" aria-expanded="false" class="u-button-reset site-nav__trigger">
+			<span aria-hidden="true" class="menu-icon">
+				 <div class="menu-icon__line"></div>
+				 <div class="menu-icon__line"></div>
+				 <div class="menu-icon__line"></div>
+			</span>
+			<p class="u-brandOrange u-marginNone"> Menu </p> 
+			{#   @feedback the menu keeps hanging off of the nav and I can not seem to position it #}
```

Comment:
So, the trigger button itself wasn't full in the menu and you can fix that by updating your `_nav.scss` in the following ways.

**Note** For these examples, I'm just going to show the bits of code I'm recommending to update; this doesn't mean you should delete anything that isn't shown -- I'll specifically call out if there's anything to delete. 😸 

1. Decrease the distance of the button from the browser's edge:

```
.site-nav {
    &__trigger {
        top: 0.9em;
        right: 0.9em;
    }
}
```

2. Change the way the margins work in the hamburger button:
```
.menu-icon {
    &__line {
//      margin: 8px auto;  <-- delete this

        &:nth-of-type(even) {  //  <-- add this to replace it
            margin-top: 8px;
            margin-bottom: 8px;
        }
    }
}
```

3. When the menu is open, there's a bit of a gap between the nav and the now visible `ul`. This is from the default margins on the `ul`. You can resolve this by adding either
- `margin: 0;` to your `u-list-reset` class (if you want all reset lists to lose this margin, but that might be unnecessarily aggressive), or
- `margin-top: 0;` to your `.site-nav__list[data-visible="true"]` mobile class.

This should sort out your mobile nav, but feel free to let me know if it's still wonky or you have questions about these changes!